lib/entry-list.ts

```diff
@@ -68,30 +68,42 @@ const validateResult = (result: OnEntryListUpdatedHandlerReturn) => {
     return
   }
 
-  throw new Error(`EntryListResult is invalid.`)
+  throw new Error(`Entry List location app data is invalid.`)
 }
 
-const schema: Record<string, (value: unknown) => boolean> = {
-  values: (value) =>
-    typeof value === 'object' &&
-    Object.keys(value as Record<string, unknown>).length > 0 &&
-    Object.values(value as Record<string, unknown>).every((item) => typeof item === 'string'),
+const schema: Record<string, (value: unknown) => void> = {
+  values: (value) => {
+    if (typeof value !== 'object') {
+      throw new Error(`Entry List location app data is invalid: 'values' should be an object.`)
+    }
+
+    const values = Object.values(value as Record<string, unknown>)
+    if (values.length === 0) {
+      // we allow passing empty values object
+      return
+    }
```

Comment:
This check is not needed as calling `.every()` on an empty array is always true

Reply:
Nice catch ☝️